source/io/CsvWriter.ooc

```diff
@@ -15,9 +15,17 @@ CsvWriter: class {
 	}
 	write: func (row: VectorList<Text>) {
 		for (i in 0 .. row count) {
-			this _fileWriter file write(row[i] toString())
+			value := TextBuilder new (row[i])
+			for (k in 0 .. row[i] count)
+				if (this _isWhitespace(row[i][k])) {
+					value prepend('\"')
+					value append('\"')
+					break
+				}
 			if (i < row count - 1)
-				this _fileWriter write(";")
+				value append(CsvReader delimiter)
+			this _fileWriter file write(value toString())
```

Comment:
```FileWriter write()``` takes either ```(Char*, SizeT)```, or ```(Char)```. I want to write the entire string without having to provide length. Is this preferable: ```this _fileWriter write(value raw, value count as SizeT)```? ```SizeT``` is as you know platform dependant.